packages/devtools_app/lib/src/shared/analytics/_analytics_stub.dart

```diff
@@ -9,8 +9,12 @@ library _analytics_stub;
 
 import 'dart:async';
 
+import 'package:logging/logging.dart';
```

Comment:
no need to add any of these logs to the stub file. This code is never hit since we ship on web